### PR TITLE
Set speed from GCS

### PR DESCRIPTION
**Pull request by mday299**
Can now set aircraft speed via a GCS command.

### CODE REVIEW

File: ArduPlane/GCS_Mavlink.cpp [hunk 1]
```diff
@@ -1266,6 +1266,12 @@ void GCS_MAVLINK::handleMessage(mavlink_message_t* msg)
 
         switch(packet.command) {
 
+        case MAV_CMD_DO_CHANGE_SPEED:
+            AP_Mission::Mission_Command cmd;
+            AP_Mission::mavlink_cmd_long_to_mission_cmd(packet, cmd);
+            plane.do_change_speed(cmd);
+            break;
```
Comment (magicrub) [inline ArduPlane/GCS_Mavlink.cpp]: you never set the result, so it will always return MAV_RESULT_UNSUPPORTED

Comment (magicrub) [inline ArduPlane/GCS_Mavlink.cpp]: Also, if it mavlink_cmd_long_to_mission_cmd does not return accepted then you should not call plane.do_change_speed()

Comment (mday299) [inline ArduPlane/GCS_Mavlink.cpp]: Good catches!  Just pushed fixes for both into this PR.

**Comment by magicrub** [comment]
merged, Thanks!

**Pull request closed without merging**